types/pi-camera/pi-camera-tests.ts

```diff
@@ -0,0 +1,41 @@
+import PiCamera from 'pi-camera';
+
+// tslint:disable-next-line no-console
```

Comment:
You could always `declare function log(msg: string)` if you just want a (statically) callable function.

Reply:
Thanks for the feedback! I've applied those changes to the test file!